package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
That sounds like it would work. Perhaps we can add `npm link /coins/halfpenny` to the post-install script if in the develop environment? Or maybe it would be better to do it as part of the test script? Either way, I don't know that halfpenny needs to be included in _package.json_ if we are doing `npm link`: it is just a test dependency, and does not need to be installed and shipped with the production or staging versions of the application.

back to you for thoughts.

@swashcap and @rsskmr: interested to know what you think here as well.